From f5031c751d3e06dd277c811c25f7f40de85f82cb Mon Sep 17 00:00:00 2001
From: Jiali Chen <chenjiali@radxa.com>
Date: Fri, 19 Dec 2025 06:01:41 +0000
Subject: [PATCH 053/268] drm: panel: jd9365da: modify timing of
 radxa,display-8hd-ad002

Compatible with amlogic series SOC MIPI DSI interface.
And tested with Radxa Dragon Q6A based on QCS6490 platform.

Signed-off-by: Jiali Chen <chenjiali@radxa.com>
---
 drivers/gpu/drm/panel/panel-jadard-jd9365da-h3.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpu/drm/panel/panel-jadard-jd9365da-h3.c
+++ b/drivers/gpu/drm/panel/panel-jadard-jd9365da-h3.c
@@ -386,12 +386,12 @@ static const struct jadard_panel_desc ra
 		.hdisplay	= 800,
 		.hsync_start	= 800 + 40,
 		.hsync_end	= 800 + 40 + 18,
-		.htotal		= 800 + 40 + 18 + 20,
+		.htotal		= 800 + 40 + 18 + 40,
 
 		.vdisplay	= 1280,
 		.vsync_start	= 1280 + 20,
 		.vsync_end	= 1280 + 20 + 4,
-		.vtotal		= 1280 + 20 + 4 + 20,
+		.vtotal		= 1280 + 20 + 4 + 30,
 
 		.width_mm	= 127,
 		.height_mm	= 199,
